xcape/cape_fortran.py

```diff
@@ -1,3 +1,13 @@
 
-def cape():
-    pass
+def cape(p_2d, t_2d, td_2d, source, ml_depth, adiabat, pinc):
+    from xcape import ALLCAPELOOP
+    # nlev has to be the first dimension
+
+    nlev, ngrid = p_2d.shape
+    
+    CAPE, CIN, MUlev, zMUlev = ALLCAPELOOP.loopcape(p_2d, t_2d, td_2d,
```

Comment:
I now have one code for all CAPE types. For MU CAPE, the routine gives as an output the # of the level where the MU CAPE is found (MUlev), and the height asl (zMUlev). 

We could have a different return statement depending on source, or just always have 4 outputs (MUlev, and zMUlev are always outputted by ALLCAPELOOP, it will be a NaN when source!=2)